src/platform/forms-system/src/js/components/ConfirmationView/ConfirmationView.jsx

```diff
@@ -75,6 +75,15 @@ export const ConfirmationView = props => {
   );
   const [submitDate, setSubmitDate] = useState(props.submitDate || null);
 
+  useEffect(
```

Comment:
Added this so if we change the incoming props (unlikely in a real scenario, but useful for simulations), then it will also update.